src/components/Item.js

```diff
@@ -75,17 +75,27 @@ const Item = ({ item, handleChange, deleteItem }) => {
         />
       </label>
 
-      <span className={className} onClick={handleModalChange}>
+      <span
+        style={{ cursor: 'pointer' }}
```

Comment:
Did this because I wasn't sure how to best updated the existing className that was based on a variable.

Reply:
I guess you’d have to add `cursor: pointer;` to all 4 of the classes where they’re declared? Or you could create another class for things that are `clickable` maybe?